_data/risc_outgoing.yml

```
@@ -77,6 +77,23 @@
     }
   }
 
+- friendly_name: MFA Limit Reached Account Locked
```

Let's expand the acronym? maybe flip the words around?

```suggestion
- friendly_name: Account Locked Due to MFA (Multi-Factor Authentication) Limit Reached
```